README.md

```diff
@@ -1,169 +1,75 @@
-# Internet Identity Service
+![](./ii-logo.png)
+
+[identity.ic0.app](https://identity.ic0.app) | [Specification][spec]
 
 [![Canister tests](https://github.com/dfinity/internet-identity/actions/workflows/canister-tests.yml/badge.svg)](https://github.com/dfinity/internet-identity/actions/workflows/canister-tests.yml)
 [![Cargo tests](https://github.com/dfinity/internet-identity/actions/workflows/cargo-tests.yml/badge.svg)](https://github.com/dfinity/internet-identity/actions/workflows/cargo-tests.yml)
 [![Frontend checks and lints](https://github.com/dfinity/internet-identity/actions/workflows/frontend-checks.yml/badge.svg)](https://github.com/dfinity/internet-identity/actions/workflows/frontend-checks.yml)
+![GitHub all releases](https://img.shields.io/github/downloads/dfinity/internet-identity/total?label=downloads&logo=github)
```

Comment:
Maybe link the releases here. Click this sens you to nirvana currently.

Reply:
Done!